### PR TITLE
chore: updated readme

### PR DESCRIPTION
**Pull request by SalOne22**
@Mikachu2333 please review

### CODE REVIEW

**Review by Mikachu2333 — CHANGES_REQUESTED** [review]

**Comment by Mikachu2333** [comment]
close #190

File: README.md [hunk 1]
```diff
@@ -62,6 +146,11 @@ For library usage check [Docs.rs](https://docs.rs/rimage/latest/rimage/)
 - Resize
 - Quantization
 
+## Known bugs
```
[the remark this answers is not in the document]
Comment (SalOne22) [reply README.md:149]: Just for safety, its better to remove all trailing slashes

**Review by Mikachu2333 — APPROVED** [review]

**Comment by Mikachu2333** [comment]
Wait for `cli.rs`

**Review by Mikachu2333 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -148,21 +148,23 @@ For library usage check [Docs.rs](https://docs.rs/rimage/latest/rimage/)
...
-    <tr>
-        <td>Antiquated</td> 
-        <td> <code>riamge png "D:\\example.jpg" -s "suf test" -d "D:\\desktop\\"</code> </td> 
-   </tr>
-    <tr>
-  		<td>Recommanded</td> 
-        <td> <code>riamge png "D:\\example.jpg" -s "suf test" -d "D:\\desktop"</code> </td> 
-    </tr>
-</table>
+- **Dir path end with `\` may cause rimage crashes** due to a cmd bug [#72653](https://github.com/rust-lang/rust/issues/72653).
```
Comment (SalOne22) [inline README.md:151]: This issue only with `\`, also this is a cmd bug

Comment (Mikachu2333) [reply README.md:151]: OK

**Comment by Mikachu2333** [comment]
Wait, what about #138 ? @SalOne22 

If not fixed, I'll add it to the known bugs subsection.

**Comment by SalOne22** [comment]
> Wait, what about #138 ? @SalOne22 

In progress. I probably release a pre-release with this issue, but in full release should be fixed.

**Review by Mikachu2333 — APPROVED** [review]

**Comment by Mikachu2333** [comment]
> In progress. I probably release a pre-release with this issue, but in full release should be fixed.

OK, so nothing should be changed now, merged.

**Pull request merged**